check_api/src/main/java/com/google/errorprone/fixes/SuggestedFixes.java

```diff
@@ -1143,9 +1143,12 @@ private static Tree suppressibleNode(TreePath path) {
       return SuggestedFix.builder()
           .replace(
               annotation,
-              annotation
-                  .toString()
-                  .replaceFirst("\\(\\)", "(" + parameterPrefix + newArgument(newValues) + ")"));
+              '@'
+                  + annotation.getAnnotationType().toString()
```

Comment:
More correct would be to do `state.getSourceForNode(annotation.getAnnotationType())`, but this method currently does not accept a `VisitorState`. Let me know if you'd like me to make this change; it appears that all callers do have a `VisitorState` in scope. (Well, in the open source repository, at least.)

Reply:
Thanks for the fix!

Adding passing in the `VisitorState` as a parameter and using `state.getSourceForNode(annotation.getAnnotationType())` SGTM if you want to make that change, but this is a clear improvement as-is, so either way is OK with me.

Reply:
Alright, I added one more commit, though arguably the improvement is minimal. (But I can imagine that at some point [`TreeToString`](https://github.com/google/error-prone/blob/master/core/src/main/java/com/google/errorprone/bugpatterns/TreeToString.java) is extended to cover non-`BugChecker` classes, in which case this change would have to be made anyway.)